jobs/user_add/templates/pre-start.sh.erb

```diff
@@ -25,10 +32,12 @@ set -ex
       end
       %>
 
-  getent passwd <%=user%> > /dev/null || useradd --create-home <%=user%>
+  getent passwd <%=user%> > /dev/null || useradd --create-home <%=user%> --base-dir "$home_dir"
```

Comment:
i think in addition to setting home we need to chown/chmod some dirs because uids may change.

Reply:
Oh you're right that may happen, probably when user order is changed for whatever reason. Thanks for pointing that out. I add that probably on Monday.